### PR TITLE
Added JoinLobbyByLobbyID()

### PR DESCRIPTION
**Pull request by 19satyum**
Now, you can join lobby using LobbyID

### CODE REVIEW

**Comment by CreepyAnt** [comment]
what this? lol

File: Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs [hunk 1]
```diff
@@ -256,6 +256,36 @@ public class EOSLobby : MonoBehaviour {
         });
     }
 
+    public virtual void JoinLobbyByID(string lobbyID)
+    {
```
Comment (FakeByte) [inline Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs]: Make sure this bracket is on the same line as the method name

File: Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs [hunk 2]
```diff
@@ -256,6 +256,36 @@ public class EOSLobby : MonoBehaviour {
...
+
+            //for each lobby found, add data to details
+            for (int i = 0; i < search.GetSearchResultCount(new LobbySearchGetSearchResultCountOptions { }); i++) {
+                LobbyDetails lobbyInformation;
+                search.CopySearchResultByIndex(new LobbySearchCopySearchResultByIndexOptions { LobbyIndex = (uint) i }, out lobbyInformation);
+                foundLobbies.Add(lobbyInformation);
+            }
+
+            if (foundLobbies.Count > 0)
+            {
```
Comment (FakeByte) [inline Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs]: Same here, make sure this bracket is on the same line as the if

**Review by FakeByte — CHANGES_REQUESTED** [review]

File: Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs (hunk 1 above)
Comment (19satyum) [reply Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs]: Fixed :+1:

**Review by FakeByte — APPROVED** [review]

**Pull request merged**